### ISSUE TITLE
Purchase a domain name

### ISSUE THREAD

**Issue by ErnieAtLYD** [codeforamerica/amiinmiami#11]
amiinmiami.org/net/com?

**Comment by ErnieAtLYD** [comment]
Was this closed because this was done or because there was no longer a priority to work on this?

**Comment by phiden** [comment]
no idea. Did I close it?

**Comment by ErnieAtLYD** [comment]
Yeah, it was closed. I remember there was some talk with Migurski but past
that I don't remember? Do you remember?

**Comment by phiden** [comment]
I don’t, no. :/

**Comment by ErnieAtLYD** [comment]
Hey @migurski, what's the process for domain name stuff again? It looks like amiinmiami.com is taken. Does anyone have objections to amiinmiamidade.com?

^ cc: @mathias-gibson (Mathias, I'm trying to tie this project up in a neat bow and handing off to brigade)
